veg2hab/main.py

```diff
@@ -173,7 +181,9 @@ def run_3_definitietabel_en_mitsen(params: ApplyDefTabelInputs):
     deftabel = DefinitieTabel.from_excel(Path(constants.DEFTABEL_PATH))
     # @Mark
     # Hier evt vertaling van hoe je override dict doorgeeft naar een Dict[str, OverrideCriterium]
-    deftabel.set_override_dict(params.override_dict)
+    # arcGIS geeft hier None als er geen override dict is meegegeven
+    if not pd.isna(params.override_dict):
```

Comment:
Ik zal de params ook even aanpassen, dat het een lege dict is dan. Maar voor nu is dit prima.